solutions/gold/cses-1670.mdx

```diff
@@ -2,9 +2,20 @@
 id: cses-1670
 source: CSES
 title: Swap Game
-author: Maggie Liu
+author: Maggie Liu, Daniel Ge
 ---
 
+<Spoiler title="Hint">
+Think of all $9!$ possible configurations of the board as the nodes, and our moves
+as the edges that connect them. We then perform BFS, starting at our intial given
+configuration.
+</Spoiler>
+
+The difficulty of this problem is with how we choose to implement the BFS. We present
```

Comment:
the difference is the way the state is hashed, right?

Reply:
yes but I didn't want to use that word since it's a later unit

Reply:
just mention & link the future module :>

Reply:
sure!